tomas/index.js

```diff
@@ -0,0 +1,84 @@
+let log = console.log;
+
+// Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range
+let x = 1;
+let y = 100;
+let result;
+if((x < 50 || x > 99) && (y < 50 || y > 99)) {
```

Comment:
Your integer variable are in range of  < 49.999 or bigger than 99! It should be in range of 50 to 99!!
You also have to check if EITHER of them are in the range not both of them!